TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/SubSystemManager.java

```diff
@@ -32,11 +32,13 @@ public class SubSystemManager {
     private static boolean leftBumperControl = false; // TODO maybe a better name?
     private static boolean xButtonControl = false; // TODO maybe a better name?
     private static boolean rightStickControl = false; // TODO maybe better name?
+    private static ElapsedTime time = new ElapsedTime();
```

Comment:
What is the use of this?

Reply:
i misread where the issue was, now im dealing with errors with pull because the branch didn't pull for a long time and it created conflictions